command/deployment_status.go

```diff
@@ -214,9 +227,10 @@ func (c *DeploymentStatusCommand) ttyMonitor(client *api.Client, deployID string
 	d.Set(spinner)
 
 	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
 
+	d.RenderFrame()
```

Comment:
Silly question, but why is RenderFrame() getting called here--does it have to do with calculating the terminal size?

Reply:
Excellent question actually. It doesn't belong here! I added it for debugging to force the rendering in main thread rather than in a goroutine!